dashboard/test/ui/step_definitions/blockly.rb

```diff
@@ -47,17 +47,49 @@
 
 Then /^block "([^"]*)" is at offset "([^"]*), ([^"]*)"$/ do |block, x, y|
   point = get_block_coordinates(get_block_id(block))
-  x.to_i.should eq point.x
-  y.to_i.should eq point.y
+  point.x.should eq x.to_i
+  point.y.should eq y.to_i
 end
 
-Then /^block "([^"]*)" is at location "([^"]*)"$/ do |block, location_identifier|
-  blockId = get_block_id(block)
-  actual_x = @browser.execute_script("return $(\"[block-id='#{blockId}']\").position().left")
-  actual_y = @browser.execute_script("return $(\"[block-id='#{blockId}']\").position().top")
+Then /^block "([^"]*)" is((?:n't| not)?) at location "([^"]*)"$/ do |block, negation, location_identifier|
+  block_id = get_block_id(block)
+  actual_x = @browser.execute_script("return $(\"[block-id='#{block_id}']\").position().left")
+  actual_y = @browser.execute_script("return $(\"[block-id='#{block_id}']\").position().top")
   location = @locations[location_identifier]
-  location.x.should eq actual_x
-  location.y.should eq actual_y
+  if negation == ''
+    "#{actual_x},#{actual_y}".should eq "#{location.x},#{location.y}"
+  else
+    "#{actual_x},#{actual_y}".should_not eq "#{location.x},#{location.y}"
```

Comment:
Likewise here the assertion was reversed; and I wanted to be able to negate this statement.